### DISCUSSION TITLE
Found a gist on setting up nebula on debian and Ubuntu

### DISCUSSION THREAD

**Discussion by skRebel** [NebulaServices/Nebula#22] [General]
https://gist.github.com/ProgrammerIn-wonderland/2f3c7d2d4b3020bc4342997067219cb9

Feel free to use this with a free VPS (I'd personally recommend oracle cloud)

**Comment by cloverdata** [comment]
This was made by Alice who is a contributor of Nebula. You can find this tutorial (slightly modified) on the main Repo :)
